src/style/fims/_record.scss

```diff
@@ -24,6 +24,10 @@
   td.sequence {
     word-break: break-all;
   }
+
+  .error {
```

Comment:
I think just using `text-danger` class is better. This keeps the error colors consistent throughout the app